### ISSUE TITLE
The time format needs to include time zone

### ISSUE THREAD

**Issue by chappjc** [decred/dcrdata#962]
Even with the fixes to https://github.com/decred/dcrdata/issues/875, the DB tables that have a TIMESTAMP type column should have this type changed to TIMESTAMPTZ to make reading/writing less error prone.  One such issue is with the /insight/api/blocks endpoint where the blocks' times do not show the time zone, and worse, are not necessarily in UTC (they are only UTC if the machine clock is in UTC).

**Comment by chappjc** [comment]
Setting 4.0 milestone and pinning it since this needs to be done for 4.0.

**Comment by xaur** [comment]
How aborting startup if the machine clock is not UTC?

**Comment by chappjc** [comment]
@xaur That's not good enough unfortunately.  The system clock would also need to be UTC (in addition to the hardware/machine/RTC clock).

For example, on an Arch Linux desktop, my hardware clock is in UTC, but my system has a timezone.  So we'd really need to abort startup if the system/OS clock is not in UTC, which I don't think is reasonable for devs.  This is good on a cloud server though.

```
$ sudo timedatectl
               Local time: Tue 2019-02-05 10:47:53 CST
           Universal time: Tue 2019-02-05 16:47:53 UTC
                 RTC time: Tue 2019-02-05 16:47:53
                Time zone: America/Chicago (CST, -0600)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no
```

**Comment by chappjc** [comment]
I'm addressing this.

**Comment by chappjc** [comment]
Resolved by PR #1002 and https://github.com/decred/dcrdata/pull/988
